.travis.yml

```diff
@@ -1,11 +1,19 @@
+env:
+  global:
+    - CC_TEST_REPORTER_ID=XXXXXXX
```

Comment:
@sseerrggii @enricostano @sauloperez guys could you please give me access to Code Climate account or send me this var? 🙏

Reply:
It can be found here :point_down: 

![Captura de pantalla 2019-03-14 a la(s) 22 27 48](https://user-images.githubusercontent.com/576701/54392743-af3d7000-46a8-11e9-96b6-f0bac6aa3040.png)